skbeam/core/tests/test_correlation.py

```diff
@@ -96,8 +98,8 @@ def test_lazy_vs_original():
     assert np.all(two_time[1] == final_gen_result_two.lag_steps)
 
 
-def test_lazy_two_time():
-    setup()
+def test_lazy_two_time(corr_setup):
+    num_levels, num_bufs, xdim, ydim, stack_size, img_stack, rois = corr_setup
```

Comment:
Why don't we call `corr_setup()` here?

Reply:
I guess the tuple of the input values is computed by 'corr_setup' fixture and passed as a parameter. It may happen that the original test was written before fixtures were available.